src/styles/views/UserPage.scss

```diff
@@ -2,7 +2,7 @@
 
 .userPage {
   .leaflet-container {
-    width: 60%;
+    width: 60vw;
     height: 60vh;
```

Comment:
@anhuwi not a square now, if you want a square change vh to vw or vw to vh

Reply:
fine with me, did not wanted a square:)